src/components/gallery/arts/art-detail.tsx

```diff
@@ -1,52 +1,22 @@
 import { useParams } from 'react-router-dom';
-import { useState } from 'react';
-import { ART_WORKS_CONTACT } from '@/constants/art-details';
+import { ART_WORKS_CONTACT } from '@/constants/gallery/art-details';
+import DetailReviews from './detail/detail-reviews';
 
 export default function ArtsDetail() {
   const { artsNo } = useParams();
 
-  const [imagePreview, setImagePreview] = useState<string | null>(null);
-  const [comment, setComment] = useState<string>('');
-  const [comments, setComments] = useState<
-    { text: string; image: string | null }[]
-  >([]);
-
-  const handleImageChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    const file = e.target.files?.[0];
-    if (file) {
-      const imageUrl = URL.createObjectURL(file);
-      setImagePreview(imageUrl);
-    }
-  };
-
-  const handleImageDelete = () => {
-    setImagePreview(null);
-  };
-
-  const handleCommentChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    setComment(e.target.value);
-  };
-
-  const handleCommentSubmit = () => {
-    if (comment.trim()) {
-      setComments([...comments, { text: comment, image: imagePreview }]);
-      setComment('');
-      setImagePreview(null);
-    }
-  };
-
   const art = ART_WORKS_CONTACT.find((item) => item.artsNo === Number(artsNo));
 
   if (!art) return <div>작품을 찾을 수 없습니다.</div>;
 
   return (
     <div className='text-center flex justify-between'>
-      <div className='md:max-w-[1080px] w-full mx-auto'>
+      <div className='md:max-w-[1080px] w-full mx-auto mt-[50px]'>
         <div className='w-full h-[78px] flex items-center pt-[40px] pb-[20px] border-b-[2px] border-b-[#DDD]'>
           <ul className='w-full flex justify-between text-[32px] font-bold '>
-            <a href='/gallery'>
-              <li>2025</li>
-            </a>
+            <li>
+              <a href='/gallery'>2025</a>
```

Comment:
```suggestion
              <Link to='/gallery'>2025</a>
```

anchor tag 대신 react-router-dom의 `Link` 메서드를 사용하는 것은 어떨까요?

![image](https://github.com/user-attachments/assets/d8b95c33-a120-4d6a-a6c7-c0782643d892)

Reply:
네! 변경하겠습니다.